man/mustache.5.ron

```diff
@@ -411,6 +411,40 @@ Can be thought of as a single, expanded template:
     {{/names}}
 
 
+### Dynamic Names
+
+Dynamic Names are a special notation to dynamically determine a tag's content.
+
+Dynamic Names consists of an asterisk, followed by a dotted name which follows
+the same notation as in an Interpolation tag. That is `{{>*dynamic}}`.
+
+This tag's dotted name, which is the Dynamic Name excluding the leading
+asterisk, references a key in the context whose value will be used in place of
+the Dynamic Name itself as content of the tag. The dotted name resolution
+produces the same value as an Interpolation tag and does not affect the context
+for further processing.
+
+Failed resolution of the dynamic name SHOULD result in nothing being rendered.
```

Comment:
This is very technical and formal compared to the rest of the manpage. Please try reformulating it more loosely in terms of what it's good for and how you can use it.

It would also be good to mention that dynamic names are an optional part of the specification. Perhaps see the lambdas subsection of the Block tags section for some inspiration.

Reply:
Okay I got it!